pkg/spec/resources.go

```diff
@@ -225,6 +252,19 @@ func (app *ControllerFields) createIngresses() ([]runtime.Object, error) {
 	return ings, nil
 }
 
+func (app *ControllerFields) createRoutes() ([]runtime.Object, error) {
```

Comment:
It would be nice to have tests for this function ;-)

Reply:
added